internal/display/display.go

```diff
@@ -75,6 +75,10 @@ func (r *Renderer) Heading(text ...string) {
 }
 
 func (r *Renderer) EmptyState(resource string) {
+	if r.Format == OutputFormatJSON {
```

Comment:
If we moved this here, do we still need the same check before calling this func in the various other files? e.g. https://github.com/auth0/auth0-cli/pull/747/files#diff-c6375d88e4e80bb65e83e7c3dd8945a7c669e86195c8e8b0d01b38e6e648e366R56

Reply:
That's certainly an option. There would be one or two instances where we'd still have to manually implement for a command's rendering function. I was trying to avoid a refactor but if we have appetite for it, I don't object.

Reply:
**Update:** Went ahead with a consolidation of this code.